### ISSUE TITLE
Build Errors on tvOS 14.5 with PVMednafen

### ISSUE THREAD

**Issue by TheDevDan** [Provenance-Emu/Provenance#1520]
Xcode 12.5
Big Sur 11.3.1

Pulled clean source directly using install from source instructions on Wiki.

Building release for tvOS 14.5

### Expected: 

Clean build

### Actual: 

Failed build

 ### Reproduce:  

See above configuration

![image](https://user-images.githubusercontent.com/20735324/118357505-847d2980-b548-11eb-8b4a-ad57291e1f1d.png)

#### APP VERSION
- [x] 1.5 *beta*
- [ ] 1.4

#### APP SOURCE
- [x] Provenance-Emu GitHub  *&nbsp;(Official)*
- [ ] Other  *&nbsp;(Unofficial)*

#### INSTALLED BY 
- [ ] [Side-Loading](https://git.io/vpYzf)
- [x] [Building](https://git.io/vpYzv) *from Source*

#### PLATFORM
- [ ] iPhone
- [ ] iPad
- [x] Apple TV

#### iOS/tvOS VERSION
- [x] 14.x
- [ ] 13.x
- [ ] 12.x
- [ ] 11.x
- [ ] 10.x

-------------------------------------------
<sup> 🚫 We **DO NOT** support unofficial builds installed from 3rd-party sites. ([Official Install](https://git.io/vxABg)) </sup><br><sup> ❓ **Need help or have a suggestion?** Join our [Official Discord](https://discord.gg/4TK7PU5) </sup><br>

**Comment by ocxo** [comment]
Experiencing same issue with same specs as described above but on Big Sur 11.4

**Comment by DanEEStar** [comment]
I have the same compiling problems...

**Comment by DanEEStar** [comment]
Update from the Discord channel. 
It seems the file `Provenance/Cores/Mednafen/mednafen/intl/VERSION` should be deleted.
It compiled then on my machine.

**Comment by DanEEStar** [comment]
Here are the more concrete steps I had to take to get a clean compile:

1. Update the Cartfile.resolved as explained below

```
diff --git a/Cartfile.resolved b/Cartfile.resolved
index 369d62ff..df407afc 100644
--- a/Cartfile.resolved
+++ b/Cartfile.resolved
@@ -1,10 +1,10 @@
 github "CocoaLumberjack/CocoaLumberjack" "3.7.0"
-github "ReactiveX/RxSwift" "5.1.1"
+github "ReactiveX/RxSwift" "5.1.2"
 github "RxSwiftCommunity/RxDataSources" "4.0.1"
 github "ZipArchive/ZipArchive" "v2.2.3"
 github "ashleymills/Reachability.swift" "v5.0.0"
 github "bcylin/QuickTableViewController" "v1.2.4"
 github "cheif/RxRealm" "a6fa47c1ba721b19a7c6da0d629bf7f0d7271d06"
-github "realm/realm-cocoa" "v5.5.0"
+github "realm/realm-cocoa" "v5.5.2"
 github "stephencelis/SQLite.swift" "0.12.2"
 github "zydeco/SteamController" "v1.2"
```

2. Delete the '/Provenance/Cores/Mednafen/mednafen/intl/VERSION' file
3. Rename thread to threads in Cores/Reicast/recast-emulator/core/linux/in prof/nixprof.cpp

**Comment by mrjschulte** [comment]
We have an inbound PR to merge the comments and findings from @DanEEStar back into the main Develop branch so everything works again on Newer macOS / Xcode builds of the Develop Branch of Provenance. We will keep this ticket open until that PR has been merged in at which point we will close this Ticket.

**Comment by paulofaria** [comment]
Hi, I was able to successfully build the project after @DanEEStar's suggestions. However, when I try to run the app, it crashes on launch with no decent error message. Does anyone have any idea what might be happening? I'm trying to run on a iPad Pro with iOS 14.6 installed.

**Comment by paulofaria** [comment]
It looks like it's an issue with Reachability. The crash is because the library is not being linked at runtime. Any idea how to solve it? I'll send the logs tomorrow.

**Comment by mrjschulte** [comment]
Any updates? We have seem this work as expected on other builds people are doing.

**Comment by JoeMatt** [comment]
Fixed in #1548
